### ISSUE TITLE
Wheeling (G19) allows illegal upgrades

### ISSUE THREAD

**Issue by byronczimmer** [Auzzy/1846-routes-web#1]
Wheeling (G19) is permitting tiles that cause track to enter grey hex Homewood (F20).

This is against track laying rules as noted in the rulebook here:
6.45 Restrictions: A tile upgrade must preserve, by both its type
(city or not) and how it is oriented, all previous track that existed
in that tile. Tiles may not be placed in the Michigan Central’s or
Ohio & Indiana’s reserved hexes until the respective company
has been purchased by a corporation or removed (either during
setup or later). Tiles cannot be placed with track that connects to:
an all-sea (blue) hexside; **either a red off-board area edge or gray
pre-printed hexside without track**; or any map edge.

Wheeling should be limited to green tile 14 (Normal City "X") and then would not be able to be upgraded further to brown.

**Comment by Auzzy** [comment]
Yeah, I came across this oversight when adding the map. It impacts a couple of spaces.
It also impacts space D18. Since I model each portion of the board as a hex (including the red off-board sections), the app doesn't know there's a gap between D18 and Windsor, so it allows you to run track into the lake.
